src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitCompoundButtonTests.cs

```diff
@@ -108,5 +108,48 @@ public void BitCompoundButtonAriaHiddenTest(bool ariaHidden, bool expectedResult
 
             Assert.AreEqual(bitCompoundButton.HasAttribute("aria-hidden"), expectedResult);
         }
+
+        [DataTestMethod,
+            DataRow("", "button", true),
+            DataRow("bing.com", "a", true),
+            DataRow("bing.com", "button", false)
+        ]
+        public void BitCompoundButtonShouldRenderExpectedElementBasedOnHref(string href, string expectedElement, bool isEnabled)
```

Comment:
please apply the logic of reaching the expected result based on input values into the test method's body and do not accept it as test input data.

Reply:
Ok I'll fix it.